org.dita-semia.resolver.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.dita-semia.resolver",
+    "description": "Resolve special attributes to standard DITA content during preprocessing. Supporting XSLT-Conref, Conbat and Advanced Key-based References.",
+    "keywords": ["custom XML", "XSLT-conref", "conbat", "advanced key-ref", "implicit key definition"],
+    "homepage": "https://github.com/dita-semia/dita-semia-resolver",
+    "vers": "0.1.2",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": "2.4"
```

Comment:
This `reg` value means the plugin works _only_ on DITA-OT 2.4. Did you mean `>=2.4`?